packages/frontend/src/config/disabled.ts

```diff
@@ -0,0 +1,95 @@
+import { ChainSlug, Slug } from '@hop-protocol/sdk'
+import capitalize from 'lodash/capitalize'
+import logger from 'src/logger'
+import { discordUrl } from 'src/utils'
+
+const emptyRoute = {
+  message: {
+    text: '',
+    href: '',
+    linkText: '',
+    postText: '',
+  },
+  source: '',
+  destination: '',
+  warningOnly: false,
+}
+
+export const ALL = 'all'
+
+export interface DisabledRoute {
+  source: ChainSlug | 'all'
+  destination: ChainSlug | 'all'
+  message: any
+  warningOnly: boolean
+}
+
+function formatDirection(source, destination) {
+  let direction: string | undefined
+  if (source === ALL) {
+    direction = `to ${capitalize(destination)}`
+  } else if (destination === ALL) {
+    direction = `from ${capitalize(destination)}`
+  } else {
+    direction = `from ${capitalize(source)} to ${capitalize(destination)}`
+  }
+  return direction
+}
+
+export function parseDisabledRoutes(
+  serializedDisabledRoutes: string,
+  serializedWarningRoutes: string
+): DisabledRoute[] {
+  const disabledRoutes = serializedDisabledRoutes?.split(',')
+  const warningRoutes = serializedWarningRoutes?.split(',')
+
+  if (disabledRoutes[0] !== '') {
+    return (disabledRoutes as any[]).map((disabledRoute, i) => {
+      const indexOfColon = disabledRoute.indexOf(':')
+      const source = disabledRoute.slice(0, indexOfColon)
+      const destination = disabledRoute.slice(indexOfColon + 1)
+
+      if (!(source in Slug || source === ALL) || !(destination in Slug || destination === ALL)) {
```

Comment:
minor suggestion: use a set here to simplify the condition

```ts
const validValues = new Set([...Object.values(Slug), ALL])
if (!(validValues.has(source) && validValues.has(destination))) {
  // error
}
```

Reply:
cool idea